gradle.properties

```diff
@@ -54,7 +54,7 @@ qosLogbackVersion=1.2.3
 swaggerJersey2JaxrsVersion=1.5.22
 slf4jApiVersion=1.7.25
 gradleGitPluginVersion=2.2.0
-k8ClientVersion=3.0.0
+k8ClientVersion=5.0.0
```

Comment:
Observing that the latest version of k8Client is 7.0.0 . ref: https://github.com/kubernetes-client/java/blob/master/README.md#compatibility
Any specific reason for not using version 7.x?

Reply:
No particular reason.. I will try to bump to 7.0. Which query still needs work?